.github/workflows/pull_request.yaml

```diff
@@ -32,9 +32,22 @@ jobs:
         compiler:
           - gcc
           - clang
+        cmake_build_type:
+          - Debug
+          - Release
+        cmake_with_testing:
+          - ON
+          - OFF
```

Comment:
I think it is enough to build release test for gcc compiler. Because the clang job's build type is debug.

Reply:
We shouldn't ignore UT for PR

Reply:
> We shouldn't ignore UT for PR

There are some error which caused by use the test/ source tree object in normal source tree.So I add the CI for build without testing source tree.

Reply:
> > We shouldn't ignore UT for PR
> 
> There are some error which caused by use the test/ source tree object in normal source tree.So I add the CI for build without testing source tree.

I think we can leave that in _**nightly**_ build.

Reply:
> > > We shouldn't ignore UT for PR
> > 
> > 
> > There are some error which caused by use the test/ source tree object in normal source tree.So I add the CI for build without testing source tree.
> 
> I think we can leave that in _**nightly**_ build.

If so, we need fix the error later.